packages/whosmysanta-backend/package.json

```diff
@@ -53,16 +26,28 @@
   "dependencies": {
     "dotenv": "^4.0.0",
     "dotenv-extended": "^1.0.4",
-    "express": "^4.15.0",
+    "express": "^4.15.2",
     "express-graphql": "^0.6.3",
     "graphql": "^0.9.1",
-    "mongoose": "^4.8.5",
-    "shortid": "^2.2.6"
+    "mongoose": "^4.8.6",
+    "shortid": "^2.2.8"
+  },
+  "scripts": {
+    "build": "babel src -d build",
+    "clean": "rimraf build",
+    "dev": "babel-node src",
+    "flow": "flow",
+    "format": "yarn run lint -- --fix && yarn run prettier",
+    "lint": "eslint . --ignore-path .gitignore",
+    "precommit": "yarn run format && lint-staged",
+    "prettier": "prettier --no-bracket-spacing --single-quote --trailing-comma all --parser flow --write *.js",
+    "start": "node build",
+    "test": "jest --coverage"
   },
   "lint-staged": {
     "*.js": [
       "yarn run prettier",
```

Comment:
We should do here the same as below with lint, eg:
`yarn run prettier -- --write`